src/components/motion/MotionController.cpp

```diff
@@ -35,8 +36,13 @@ namespace {
   }
 }
 
-void MotionController::Update(int16_t x, int16_t y, int16_t z, uint32_t nbSteps) {
-  if (this->nbSteps != nbSteps && service != nullptr) {
+void MotionController::AdvanceDay() {
+  --nbSteps;           // Higher index = further in the past
+  nbSteps[today] = -1; // Ensure that the `this->nbSteps[today] != nbSteps` condition in `Update()` will be FALSE
```

Comment:
I think it'd be better to assign zero here and do service->OnNewStepCount() inside this function. Reasoning about -1 applied to an unsigned int is complex and so is reasoning about the delta steps calculation

Reply:
Done